LambdaEngine/Source/Rendering/ImGuiRenderer.cpp

```diff
@@ -302,10 +302,14 @@ namespace LambdaEngine
 		uint32 modFrameIndex,
 		uint32 backBufferIndex,
 		CommandList** ppFirstExecutionStage,
-		CommandList** ppSecondaryExecutionStage)
+		CommandList** ppSecondaryExecutionStage,
+		bool Sleeping)
```

Comment:
Functions use pascalcase. Parameters and local variables use camelcase. `bool Sleeping` -> `bool sleeping`

Reply:
Done